src/learner.Rmd

```diff
@@ -111,6 +111,21 @@ and defaults of a learning method without explicitly constructing it (by calling
 getParamSet("classif.randomForest")
 ```
 
+Functions for accessing Learner's meta information are available in [%mlr]. We can use ``getLearnerId``,``getLearnerShortName`` and  ``getLearnerType`` to get Learner's id, short name and class, respectively. Moreover, in order to show the required packages for the Learner, one can call ``getLearnerPackages``.
```

Comment:
* "Functions for accessing **a** Learner's ..."
* Could you please link to the documentation of these functions, i.e., [&getLearnerId], [&getLearnerShortName] etc.?